frontend/src/store/account.js

```diff
@@ -213,6 +213,13 @@ const actions = {
                     hash: router.currentRoute.value.hash
                 })
             }
+            if (settings.license?.expired) {
```

Comment:
If #1744 concludes that everything keeps running, then we shouldn't force admin to the license page and nothing else, otherwise they won't be able to administer the running platform.